components/ILIAS/UI/src/Implementation/Component/Layout/Page/Renderer.php

```diff
@@ -115,6 +116,22 @@ protected function renderStandardPage(
         return $tpl->get();
     }
 
+    protected function renderMailPage(
+        Component\Layout\Page\Mail $component,
```

Comment:
Please change this to `Mail` as well, this way you will get access to the implementation details.